pkg/agent/resourcemanager/reporter/cnr/cnrreporter.go

```diff
@@ -61,6 +61,8 @@ const (
 type cnrReporterImpl struct {
 	cnrName string
 
+	// labels contains the default config for CNR created by reporter
+	labels map[string]string
```

Comment:
Would it be better to name it defaultLabels?

Reply:
yep, done